UnrealThirdPersonMP/README.md

```diff
@@ -0,0 +1,100 @@
+what folder you want this file to be saved to. 
+
+![showing the way to set the build target to the project and package the game client](images/PackageGame.PNG)
+
+The packaged game client will be in a folder called "WindowsNoEditor".
+
+### Testing Locally using LocalMultiplayerAgent
+
+The following instructions will be for container mode. Because of this, we first need to ensure that our zipped game server has all required system DLLs that are not included in the container image. 
+
+To check this, you need to determine the required DLLs by running the ListDLLs tool that is [found here.](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/determining-required-dlls)
+
+Once you have included all the necessary DLLs, follow the instructions to set up the [Local Multiplayer Agent](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/locally-debugging-game-servers-and-integration-with-playfab) release that you downloaded and configure the MultiplayerSettings file by following the ***instructions for running your game server in container mode.***
+
+Also, please ensure that the start command for your game server is running the executable found in the binaries folder, not the root file which I discovered through [this Playfab Community](https://community.playfab.com/questions/53596/localvmagent-not-working-while-running-container-m.html) post. 
+
+```
+eg: C:\Assets\YourGame\Binaries\Win64\YourGameServer.exe
+```
+
+Then when your game server is running through LocalMultiPlayerAgent and has a status of *Active*, you can run your client from the command line with a specific IP address and port as arguments. For example, this command:
+
+```
+.\GameClient.exe 127.0.0.1 -port=30000
+```
+
+would mean that you are running your game client executable, GameClient.exe, to connect to your game server that is on 127.0.0.1 (i.e. local host) through port 30000. You can learn more about [command line arguments here](https://docs.unrealengine.com/4.26/en-US/ProductionPipelines/CommandLineArguments/).
+
+### Testing using Playfab MPSAllocatorSample
+
+With your [developer.playfab.com](developer.playfab.com) account, create a new TitleID or use an existing one.
+
+Click on this new TitleID, and go the Multiplayer tab under the Build section.
+
+![Screenshot of what the Multiplayer Server tab looks like](images/MultiplayerServersTab.png)
+
+Create a new build and make sure that the Server type is "Container". Upload your zipped up game server along with all of the DLLs (the same one you used with LocalMultiplayerAgent) and make sure the mount path and start command are correct.
+
+Set the port to be Unreal's port number, 7777.
+
+Then choose whichever regions you would like and create the new build. 
+
+Give the build a few minutes to be deployed.
+
+Once your build is deployed, make sure that you have the MpsSamples repository cloned on your desktop. 
```

Comment:
Can we add a note here that the "server" part of the sample is done and we're gonna work on the client now?

Reply:
something like "Build is deployed - now let's see how we can use the MpsAllocatorSample to allocate a game server so that clients can connect"

Reply:
you can add this [https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/allocating-game-servers-and-configuring-vs-debugging-tools](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/allocating-game-servers-and-configuring-vs-debugging-tools) and this [https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/connecting-clients-to-game-servers](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/connecting-clients-to-game-servers)